### PR TITLE
DataFusion expr conversion

### PR DESCRIPTION
**Pull request by gatesn**

### CODE REVIEW

File: vortex-expr/src/datafusion.rs [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#![cfg(feature = "datafusion")]
```
Comment (a10y) [inline vortex-expr/src/datafusion.rs:1]: does this actually work or does this only flag the one import statement?

should this go in the module declaration instead?

Comment (gatesn) [reply vortex-expr/src/datafusion.rs:1]: The exclamation mark means it applies to the file

Comment (a10y) [reply vortex-expr/src/datafusion.rs:1]: nvm apparently it works

**Review by a10y — APPROVED** [review]
LGTM, we should probably start running CI with features enabled if we aren't already

**Comment by a10y** [comment]
actually looks like we are already!

**Pull request merged**